dnsprovider/pkg/dnsprovider/providers/do/dns.go

```diff
@@ -185,20 +185,27 @@ func (r *resourceRecordSets) List() ([]dnsprovider.ResourceRecordSet, error) {
 		return nil, err
 	}
 
-	var rrset *resourceRecordSet
 	var rrsets []dnsprovider.ResourceRecordSet
+	rrsetWithoutDups := make(map[string]*resourceRecordSet)
```

Comment:
Should this be plural to keep similar with `rrsets` above?
```suggestion
	rrsetsWithoutDups := make(map[string]*resourceRecordSet)
```

Reply:
sure

Reply:
Thanks. Please also squash the commits and change the commit message to something explaining the change, like "[DigitalOcean]: accept multiple records in dns-controller".
/lgtm

Reply:
Done.

Reply:
👍